x-pack/plugin/shutdown/src/main/java/org/elasticsearch/xpack/shutdown/TransportGetShutdownStatusAction.java

```diff
@@ -338,6 +353,26 @@ static ShutdownShardMigrationStatus shardMigrationStatus(
                 ),
                 decision
             );
+        } else if (relocatingShards == 0 && initializingShards == 0 && temporarilyUnmovableShard.isPresent()) {
```

Comment:
Actually I wonder if the condition should be `initializingShards == 0 && relocatingShards == 0 && startedShards == temporarilyUnmovableShards.size()`.

Meaning all shards are not moving because the are all throttled, not just before we were lucky to see the intermediate state with no movements and a single throttled shard. WDYT?